orga/translations/fr.json

```
@@ -87,6 +87,17 @@
       },
       "loader": "Chargement des proportions par statut",
       "title": "Statuts"
+    },
+    "participants-count": {
+      "information": "Retrouvez ici le nombre de participations totales à votre campagne. Ce chiffre comprend l’ensemble des participants ayant saisis le code et commencé son parcours ou son envoi profil.",
```

À la lecture, la phrase me semble bizarre, je propose une petite modification
```suggestion
      "information": "Retrouvez ici le nombre de participations totales de votre campagne. Ce chiffre comprend l’ensemble des participants ayant saisis le code et commencé son parcours ou son envoi profil.",
```